gwf/cli/__main__.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import absolute_import, print_function
```

Comment:
I think this file should be called `entrypoint.py` to make it clear that this is the entry point of the program.